jscomp/test/equal_exception_test.js

```diff
@@ -91,23 +89,17 @@ function is_normal_exception(_x) {
     _1: 3
   };
   try {
-    throw new Error(v.RE_EXN_ID, {
-          cause: v
-        });
+    throw v;
   }
   catch (raw_exn){
     let exn = Caml_js_exceptions.internalToOCamlException(raw_exn);
     if (exn.RE_EXN_ID === A) {
       if (exn._1 === 3) {
         return;
       }
```

Comment:
I see it worth renaming to:
```suggestion
  catch (exn) {
    let exnCause = Caml_js_exceptions.internalExnToCause(exn);
    if (exnCause.RE_EXN_ID === A) {
      if (exnCause._1 === 3) {
        return;
      }
      throw exn
```